src/fictionalpancake/turbospork/Node.java

```diff
@@ -94,6 +106,9 @@ public void updateProp(String key, String value) {
             case "owner":
                 owner = Integer.parseInt(value);
                 break;
+            case "stasis":
+                setFrozen(value.equals("true"));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork/file/2570184727/issues/source?bid=3248828&fileBranchId=3297451#l110)